public/css/index.css

```diff
@@ -505,7 +505,7 @@ footer .ui.left,footer .ui.right{line-height:40px}
 .repository.file.list .non-diff-file-content .view-raw img{padding:5px 5px 0 5px}
 .repository.file.list .non-diff-file-content .plain-text{padding:1em 2em 1em 2em}
 .repository.file.list .non-diff-file-content .plain-text pre{word-break:break-word;white-space:pre-wrap}
-.repository.file.list .non-diff-file-content .csv{overflow-x:auto}
+.repository.file.list .non-diff-file-content .csv{overflow-x:auto;padding:0!important}
```

Comment:
This change seems extra. Why is it needed ?

Reply:
As the `.markdown` class is now added for CSV files, we want to reset the padding, restoring the old behaviour of `.csv` class only. This is only needed for CSV, as this class was introduced with the same PR that removed the `.markdown` class

Reply:
Ok it feel that we should introduce a new common class containing markdown style except padding and let only markdown specific inside .markdown but this would be good enough.